README.md

```diff
@@ -24,6 +24,12 @@ image to fulfill CNF Test dependency requirements:
 If you are using a different test partner pod, please ensure you provide these or the image will be unable to run all CNF
 Certification tests.
 
+## test-parner deployment label
```

Comment:
partner

Reply:
Done in commit [e536b36](https://github.com/test-network-function/cnf-certification-test-partner/pull/30/commits/e536b362136aff1adb5d9a74f186522f43a301ec)